help/manage-user-stream-subscriptions.md

```diff
@@ -33,6 +35,17 @@ other users to a stream][configure-invites].
 
 1. Click **Add**.
 
+{tab|via-user-profile}
+
+{!right-sidebar-view-profile.md!}
+
+1. Select the **Streams** tab.
+
+1. Under **Subscribe user to streams**, select the stream you would like
+   to add the user to.
```

Comment:
I think we should follow a similar pattern to https://zulip.com/help/starting-a-new-topic#how-to-start-a-new-topic (**From other views** tab), and say something like:

```suggestion
1. Under **Subscribe user to streams**, select a stream from the
   dropdown list. You can start typing to filter streams.
```